docs/developing.md

```diff
@@ -67,37 +67,36 @@ git clone https://github.com/openshift/hive.git
+[Kind](https://github.com/kubernetes-sigs/kind) can be used as a lightweight development environment for deploying and testing Hive. Currently, we support 0.8.1 version of Kind. The following instructions cover creating an insecure local registry (allowing for dramatically faster push/pull), and configuring your host OS, as well as the kind cluster to access it. This approch runs Hive in a container as you would in production, giving you the best coverage for manual testing.
 
 This approach requires [Docker](https://docs.docker.com/install). At present we do not have kind working with podman.
 
-Deploy a local insecure registry container, and configure your host docker daemon to be able to use it:
+If you encounter ImagePullErrors, and your kind container cannot reach your registry container, you may be experiencing problems with Fedora (at least 32, possibly earlier as well) and Docker. You can attempt to work around this by changing the FirewallBackend in the /etc/firewalld/firewalld.conf file from nftables to iptables and restarting docker. (see [this comment](https://github.com/kubernetes-sigs/kind/issues/1547#issuecomment-623756313)
 
-```bash
-./hack/create-insecure-registry.sh
-```
 
-Create a kind cluster named 'hive' to deploy to. You can create as many kind clusters as you need.:
+Create a local insecure registry (if one does not already exist) and then a kind cluster named 'hive' to deploy to. You can create additional clusters if desired by provising a different name argument to the script.
 
 ```bash
 ./hack/create-kind-cluster.sh hive
 ```
 
-`docker ps` should now show you a "registry" and a "hive" container running.
+NOTE:  The following error will occur the first time you create your registry container, it is harmless and can be safely ignored.
 
-```bash
-$ docker ps
-CONTAINER ID        IMAGE                  COMMAND                  CREATED             STATUS              PORTS                                  NAMES
-2756e565065a        kindest/node:v1.15.3   "/usr/local/bin/entr…"   29 hours ago        Up 29 hours         40393/tcp, 127.0.0.1:40393->6443/tcp   hive-control-plane
-1dc8a3c59d84        registry:2             "/entrypoint.sh /etc…"   2 weeks ago         Up 8 days           0.0.0.0:5000->5000/tcp                 registry
+```
+Error response from daemon: container fcca36a26da5601d6453c0c53ab5909eb6ca8ffe42de0f8634dd7213f107cef0 is not connected to network kind
 ```
 
-Configure kubectl/oc to talk to your new cluster:
+`docker ps` should now show you a "registry" and a "hive" container running.
 
 ```bash
-export KUBECONFIG="$(kind get kubeconfig-path --name="hive")"
```

Comment:
The new kind does not have `get kubeconfig-path` so also need to change the docs in Adopting clusterdeployments section

Reply:
Thanks, I replaced it with /path/to/admin/kubeconfig.